### PR TITLE
Add two new subprojects to SIG AWS

### PR DESCRIPTION
**Pull request by d-nishi**
Add two new subprojects to SIG AWS
Autogenerate docs

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -527,12 +527,10 @@ sigs:
...
       - name: Justin Santa Barbara
         github: justinsb
-      - name: Kris Nova
-        github: kris-nova
-        company: VMware
       - name: Nishi Davidson
         github: d-nishi
-        company: AWS
+      - name: Kris Nova
+        github: kris-nova
```
Comment (nikhita) [inline sigs.yaml]: Should Kris' affiliation be maintained?

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -23,8 +23,8 @@ aliases:
     - mwielgus
   sig-aws-leads:
     - justinsb
-    - kris-nova
     - d-nishi
+    - kris-nova
```
Comment (nikhita) [inline OWNERS_ALIASES]: This change isn't really needed (its a no-op)

File: sigs.yaml (hunk 1 above)
Comment (nikhita) [reply sigs.yaml]: Also... could you move this change to a separate commit since it's unrelated to adding subprojects? :grimacing:

Comment (d-nishi) [reply sigs.yaml]: this was wrong as Kris was linked to Heptio. I made it consistent with other co-chairs.

I have it in the commit history that the PRs make 2 changes. Its such a small change - do I really need to separate them?

**Pull request closed without merging**